packages/main/src/themes/Toolbar.css

```diff
@@ -0,0 +1,88 @@
+:host(:not([hidden])) {
+	display: inline-block;
+	width: 100%;
+	height: 2.75rem;
+	background: var(--ui5-toolbar-Background);
+	border-bottom: 0.0625rem solid var(--sapGroup_ContentBorderColor);
+	flex-shrink: 1;
+	color: var(--sapButton_TextColor);
+}
+
+:host(:not([hidden])[data-ui5-compact-size]) {
+	height: 2rem;
```

Comment:
This should be handled via parameter (similar to -_ui5-toolbar-separator-height) because this would work only if data-ui5-compact-size is set on the host, but it could be set on a different DOM element.
 And second, you miss the other 2 options for setting compact - via the  .ui5-content-density-compact, .sapUiSizeCompact classes